scalardb-analytics-spark-sample/sql/postgres_copy.sql

```diff
@@ -0,0 +1,13 @@
+create schema sample_ns;
```

Comment:
Is there any instruction to execute this SQL file?

Reply:
No. But, the `postgres` image automatically executes any `.sql` and `.sh` in `/docker-entrypoint-initdb.d`, so we just mount this file under that directory.

Reply:
>  the postgres image automatically executes any .sql and .sh in /docker-entrypoint-initdb.d, so we just mount this file under that directory.

Oh, I didn't know that. Sounds good!

Reply:
JFYI, refer to the "Initialization scripts" section on the image description. https://hub.docker.com/_/postgres

AFAIK, several database docker images have similar functionality.